manager/integration/tests/test_basic.py

```diff
@@ -4874,11 +4874,12 @@ def backup_failed_cleanup(client, core_api, volume_name, volume_size,  # NOQA
     vol.snapshotBackup(name=snap.name)
 
     # write some data to the volume
-    data = {
-        'pos': 0,
-        'content': common.generate_random_data(volume_size),
-    }
-    write_volume_data(vol, data)
+    volume_endpoint = get_volume_endpoint(vol)
+
+    # Take snapshots4 without overlapping
+    data_size = volume_size/Mi
+    write_volume_dev_random_mb_data(
+        volume_endpoint, 0, data_size)
```

Comment:
NIT: any difference here...?

Reply:
`generate_random_data` is slow when creating a big file, so use `/dev/urandome` instead.

Size is updated here.
```
backup_name = backup_failed_cleanup(client, core_api, volume_name, 1024*Mi)
```